src/commands/profiles/list.js

```diff
@@ -16,7 +16,7 @@ class ProfilesList extends BaseCommand {
       this.userConfig.projects.unshift(strippedEnvProfile);
       this.userConfig.setActiveProfile(strippedEnvProfile.id);
     }
-    const profiles = this.userConfig.projects;
+    const profiles = this.userConfig.projects.slice(0);
```

Comment:
Why's slicing needed? Might be interpreting it wrong, but isn't `Array.slice(0)` kinda equal to referencing the same? Or does it require deep-cloning?

Reply:
I used slice to create a shallow copy. It requires shallow copy because [`getActiveProfile`](https://github.com/twilio/twilio-cli/blob/release-feature-branch/src/commands/profiles/list.js#L32), uses the `projects` to return the active profile.